packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -480,6 +492,10 @@ - (void)keyDown:(NSEvent *)event
 		[[self window] makeFirstResponder:viewToFocus];
 		[viewToFocus scrollRectToVisible:[viewToFocus bounds]];
 	}
+	else if (viewToFocus == nil)
```

Comment:
why does this empty block exisit? it can't be else if (viewToFocus != nil) { [super keydown]} ?

Reply:
It's intentional to not call `[super keydown]` for when `vieweToFocus == nil` returns true because then it'd move focus to the next menu item (in the context menu scenario).

Reply:
isn't that the exact same logic i proposed?

Reply:
Followed through offline, addressed in 74c28db9c230ef16aa8b9d24dd5c8662d0399795